workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -89,7 +94,7 @@ public void DisposeAllRequiredFields(MonoBehaviour behaviour)
                             .WithField("Field", field.Name));
                     }
 
-                    requirableToBeDisposed.Dispose();
+                    requirableToBeDisposed?.Dispose();
```

Comment:
I think in this case it may be better if the `== null` branch causes a `continue` instead